README.md

```diff
@@ -1,7 +1,7 @@
-[![Build Status](https://travis-ci.org/wcxf/wcxf-python.svg?branch=master)](https://travis-ci.org/wcxf/wcxf-python) [![Coverage Status](https://coveralls.io/repos/github/wcxf/wcxf-python/badge.svg?branch=master)](https://coveralls.io/github/wcxf/wcxf-python?branch=master)
-
 # WCxf Python API and command line interface
 
+**Note: the wcxf package has been merged into [wilson](https://github.com/wilson-eft/wilson) and is deprecated as a standalone package.**
+
 This Python package provides a Python API and command line interface to
```

Comment:
Since the command line interface is now provided by `wilson`, maybe the readme should be changed accordingly. I would suggest to remove the section on the command line interface from `wcxf` and to add it in the `wilson` readme. In the `wcxf` readme, one can then mention that the command line interface is provided by `wilson` and that it is described in the readme of `wilson`.

Reply:
Right, I will add it. We should also note that this changed, so people don't get confused (e.g. when looking at the WCxf paper).

Reply:
On second thought: isn't it fine as it is? The added comment makes clear that one shouldn't use the package itself (so the "installation" section is clearly marked as obsolete), but the CLI actually didn't change, since `wilson` still provides the console entry point `wcxf`.